rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -1040,8 +1044,64 @@ void registerBtcTransaction_whenSpendTransactionHashIsNotSaved_shouldNotProcessS
             bridgeStorageProvider.save();
 
             // assert
-            assertActiveFederationUtxosSize(activeFederationUtxosSizeBeforeRegisteringTx);
-            assertTransactionWasNotProcessed(svpSpendTransaction.getHash());
+            // tx registration
+            assertActiveFederationUtxosSize(activeFedUtxosBeforeRegisteringTx.size() + 1);
+            assertTransactionWasProcessed(svpSpendTransaction.getHash());
+            assertNoSvpSpendTxHash();
+
+            // svp success
+            assertNoProposedFederation();
+
+            assertNewActiveFederationCreationBlockHeightWasSet();
+            assertLastRetiredFederationScriptWasSet();
+
+            assertNewAndOldFederationsWereSet();
+
+            int outputSentToActiveFedIndex = 0;
+            TransactionOutput spendTxOutputSentToActiveFed = svpSpendTransaction.getOutput(outputSentToActiveFedIndex);
+            UTXO utxoSentToActiveFederation = new UTXO(
+                svpSpendTransaction.getHash(),
+                outputSentToActiveFedIndex,
+                spendTxOutputSentToActiveFed.getValue(),
+                0,
+                svpSpendTransaction.isCoinBase(),
+                spendTxOutputSentToActiveFed.getScriptPubKey()
+            );
+            activeFedUtxosBeforeRegisteringTx.add(utxoSentToActiveFederation);
+            assertUTXOsWereMovedFromNewToOldFederation(activeFedUtxosBeforeRegisteringTx);
```

Comment:
not sure if this should be being tested here too. We are already doing it in `VoteFederationChangeTest`. Maybe we could just test that `fedSupport.commitProposedFederation` is being called